content/advanced/340_appmesh_flagger/flagger_setup.md

```diff
@@ -0,0 +1,123 @@
+---
+title: "Flagger Set Up"
+date: 2021-03-18T00:00:00-05:00
+weight: 10
+draft: false
+---
+
+#### PreRequisite
```

Comment:
May be Prerequisites instead PreRequisite?

Reply:
Done